johnsonandjohnson/Bodiless-JS
# fix(sites): remove set-missing-symlinks from site prepare during lerna bootstrap.

## Changes


- @bodiless namespace no longer available for site level symlink during lerna bootstrap. Having this `prepare` blocks `npm install` on site level.



## Test Instructions


1. Setup and build the project
2. Light regression test on test/demo site.
3. Compare the build with main branch build.

Expected
- no tailwindcss config change
- no doc build change
- env .env.* changes.


## Related Issues
